packages/component-base/test/typings/data-provider-controller/i18n-mixin.types.ts

```diff
@@ -0,0 +1,17 @@
+import { I18nMixin } from '../../../src/i18n-mixin.js';
```

Comment:
This file should be probably placed outside of the `data-provider-controller` folder.